geograypher/meshes/meshes.py

```diff
@@ -851,6 +861,17 @@ def get_values_for_verts_from_vector(
             return labeled_verts, all_values
         # Else return a dict of all requested values
         return labeled_verts_dict, all_values_dict
+    
+    def save_IDs_to_labels(self, savepath: PATH_TYPE):
```

Comment:
I'd add a docstring to this. Also, if I read this, I'd assume that I'd provide a full filepath and the result would be saved there verbatim. I'd consider this approach for greater flexibility, and let the calling methods make the determination of the exact output format. But that's up to you.

Reply:
In any case, the docstring should reflect precisely what input is expected. And I'd consider renaming to `savefolder` or similar if you want to keep the existing logic.